graphene_sqlalchemy/types.py

```diff
@@ -284,6 +319,11 @@ def __init_subclass_with_meta__(
     def is_type_of(cls, root, info):
         if isinstance(root, cls):
             return True
+        if isawaitable(root):
```

Comment:
I've been using this in dev and noticed that there actually is a case where `is_type_of` is called with an awaitable `root`:

Consider
```python
class MyModel(Base):
    __tablename__ = "Hi"
    id = Column(Integer(), primary_key=True)
    foo = Column(String())


class MyModelNode(SQLAlchemyObjectType):
    class Meta:
        model = MyModel
        interfaces = (relay.Node,)

class Query(graphene.ObjectType):
    test = graphene.Field(MyModelNode)
        
    @classmethod
     async def resolve_test(parent, info):
         await asyncio.sleep(0.1)
       return MyModel(id=1,foo="Test")
```
we will end up with the exception. So I think another round of `isawaitable` -> `return async def...` is necessary here. What do you think?

Reply:
Nevermind, there was another issue with boilerplate code. Everything can remain as is.